### DISCUSSION TITLE
Field is in deferred_load mode, so can't be accessed.

### DISCUSSION THREAD

**Discussion by mollyjames2** [Parcels-code/Parcels#1456] [Q&A]
Hi there, I'm trying to plot my u and v fields. I understand that fieldset.show() is no longer supported so I am using plt.colormesh as per the NEMO tutorial but I'm getting the Error message listed in the title - I've tried googling my way around this but it just takes me back to the source code here. My code is below:

```fieldset = FieldSet.from_nemo(filenames, variables, dimensions), allow_time_extrapolation = True)

plt.pcolormesh(fieldset.U.grid.lon, fieldset.U.grid.lat, fieldset.U.data[0,0,:,:])
plt.show()

plt.pcolormesh(fieldset.V.grid.lon, fieldset.V.grid.lat, fieldset.U.data[0,0,:,:])
plt.show()
```

the error message reads as follows:

``` plt.pcolormesh(fieldset.U.grid.lon, fieldset.U.grid.lat, fieldset.U.data[0,0,:,:])
plt.show()

plt.pcolormesh(fieldset.V.grid.lon, fieldset.V.grid.lat, fieldset.U.data[0,0,:,:])
plt.show()
```

**Comment by erikvansebille** [comment]
Ah, this is a good point. You will need to issue
```python
fieldset.computeTimeChunk(0,1)
```

See also the API at https://docs.oceanparcels.org/en/latest/reference/fields.html#parcels.fieldset.FieldSet.computeTimeChunk
